service/src/test/java/uk/nhs/adaptors/oneoneone/cda/report/mapper/OrganizationMapperTest.java

```diff
@@ -52,11 +61,54 @@
     @Mock
     private II ii;
 
+    @Mock
+    private POCDMT000002UK01InformationRecipient informationRecipient;
+
+    @Mock
+    private POCDMT000002UK01IntendedRecipient intendedRecipient;
+
+    @Mock
+    private Node node;
+
+    @Mock
+    private ON on;
+
     @Mock
     private ResourceUtil resourceUtil;
 
+    @Test
+    public void shouldMapOrganizationInformationRecipient() {
+
+        POCDMT000002UK01Organization itkOrganization = mockItkOrganization();
+
+        when(informationRecipient.getIntendedRecipient()).thenReturn(intendedRecipient);
+        when(intendedRecipient.getReceivedOrganization()).thenReturn(itkOrganization);
+        when(intendedRecipient.isSetReceivedOrganization()).thenReturn(true);
+        lenient().when(informationRecipient.getTypeCode()).thenReturn(forString(PRCP_TYPE_CODE));
+        when(itkOrganization.getNameArray(0)).thenReturn(on);
+        when(nodeUtil.getNodeValueString(on)).thenReturn(HEALTHCARE_SERVICE_NAME);
+
+        Organization organization = organizationMapper.mapOrganization(informationRecipient);
+
+        assertThat(organization.getType().get(0).getCoding().get(0).getCode()).isEqualTo(PRCP_TYPE_CODE);
```

Comment:
Maybe we could use getTypeFirstRep() and getCodingFirstRep() methods instead of accessing items by index 0?